core/mixins/_media-queries.scss

```diff
@@ -97,3 +105,16 @@
     @content;
   }
 }
+
+
+/**
+ * Retina.
+ */
+
+@mixin respond-retina($resolution: $retina-resolution) {
```

Comment:
I'm thinking how about the name `respond-resolution` instead seeing as that's the actual media feature this mixin uses? I don't know I can see the case for both, I'm just not a huge fan of using terms invented by companies, in this case Apple.